src/guides/v2.3/release-notes/backward-incompatible-changes/reference.md

```diff
@@ -6,10 +6,24 @@ redirect_from:
 - /guides/v2.3/release-notes/backward-incompatible-changes/open-source.html
 ---
 
-The changes are aggregated into two tables:
+Use this page to review high-level reference information for all backward incompatible changes in each release. Backward incompatible changes that have a major impact and require detailed explanation and special instructions are documented in the [Backward incompatible changes highlights]({{page.baseurl}}/release-notes/backward-incompatible-changes/index.html) topic.
 
-1. **Changes in classes** that contains backward incompatible changes made to the PHP classes
-1. **Changes in interfaces** that contains backward incompatible changes made to the PHP interfaces
+We use a custom tool that extends a PHP semantic version checker to auto-generate this content. The tool compares the code base from a previous release with the code base from the latest release. Backward incompatible changes for each release are aggregated into the following tables (if applicable):
+
+-  Class changes
+-  Class API membership changes
+-  Interface changes
+-  System changes
+-  XSD changes
```

Comment:
Also, DI changes, Layout changes, Database changes
see https://htmlpreview.github.io/?https://github.com/magento/devdocs/blob/2.4.0-develop/src/_includes/backward-incompatible-changes/open-source/2.3.0-2.4-develop.html

Reply:
It makes sense to mention that the list of different types of backward incompatible changes was extended after 2.3.5 release. Otherwise, we need to regenerate BICs for 2.3.0-2.3.4 releases, that is not a problem.

Reply:
I'll add a note clarifying the difference. I don't think we need to go back and rerun each release.